buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -95,7 +90,7 @@ class BeamModulePlugin implements Plugin<Project> {
     boolean ignoreRawtypeErrors = false
 
     /** Controls whether the dependency analysis plugin is enabled. */
-    boolean enableStrictDependencies = false
+    boolean enableStrictDependencies = true
```

Comment:
enabled dependency check via wfhartford plugin

Reply:
Great. Once it is enabled we can delete the option, too.